.github/workflows/script/models/cpp_graph_inference.sh

```diff
@@ -44,7 +44,7 @@ function main() {
         infer_cmd="./build/bin/run_gptj"
         model_name="EleutherAI/gpt-j-6b"
         input_model="/tf_dataset2/models/pytorch/gpt-j-6B"
-        precision_list=("q4_j_b128")
+        precision_list=("q4_j_b128", "q4_j_b128_asym")
```

Comment:
do we need q4 j b32 asym  test?

Reply:
I think more configuration better put under extension tests?